backend/src/core/loaders/express.loader.ts

```diff
@@ -71,7 +71,13 @@ const expressApp = ({ app }: { app: express.Application }): void => {
   // in the parseEvent() handle before parsing the SES event.
   app.use('/v1/callback/email', express.text({ type: 'application/json' }))
 
-  app.use(express.json())
+  app.use(
+    express.json({
+      // this must be significantly bigger than transactionalEmail.bodySizeLimit so that users who exceed limit
+      // will get 404 error informing them of the size of the limit, instead of 500 error
+      limit: config.get('transactionalEmail.bodySizeLimit') * 10,
+    })
+  )
   app.use(express.urlencoded({ extended: false }))
```

Comment:
Actually they can still use urlencoded format to send a request larger than `bodySizeLimit` actually

Reply:
ohh I didn't realise that — let me investigate this later and replicate on my API client

Reply:
OK replicated and fixed.